src/lib/datepicker/datepicker.ts

```diff
@@ -29,21 +30,42 @@ import {MdDatepickerInput} from './datepicker-input';
 import {CalendarLocale} from '../core/datetime/calendar-locale';
 import 'rxjs/add/operator/first';
 import {Subscription} from 'rxjs/Subscription';
+import {MdDialogConfig} from '../dialog/dialog-config';
 
 
 /** Used to generate a unique ID for each datepicker instance. */
 let datepickerUid = 0;
 
 
-/** Component responsible for managing the datepicker popup/dialog. */
+/**
+ * Component used as the content for the datepicker dialog and popup.
+ * @docs-internal
+ */
 @Component({
   moduleId: module.id,
-  selector: 'md-datepicker, mat-datepicker',
-  templateUrl: 'datepicker.html',
-  styleUrls: ['datepicker.css'],
+  selector: 'md-datepicker-content',
+  templateUrl: 'datepicker-content.html',
+  styleUrls: ['datepicker-content.css'],
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
+export class MdDatepickerContent implements AfterContentInit {
+  datepicker: MdDatepicker;
+
+  constructor(private _elementRef: ElementRef) {}
+
+  ngAfterContentInit() {
+    this._elementRef.nativeElement.querySelector('.mat-calendar-body').focus();
+  }
+}
+
+
+/** Component responsible for managing the datepicker popup/dialog. */
+@Component({
+  moduleId: module.id,
+  selector: 'md-datepicker, mat-datepicker',
+  template: '',
```

Comment:
The reason I made it a component is because in order to use it users need to do something like:
```
<input [mdDatepicker]="d'>
<md-datepicker #d></md-datepicker>
```

If I used a directive they had to do:
```
<input [mdDatepicker]="d'>
<md-datepicker #d="mdDatepicker"></md-datepicker>
```

which felt like a lot of "datepicker"